AMM/SPECIFICATION.md

```diff
@@ -135,41 +133,63 @@ This sub-section details what a user is able to do, e.g., click a button and "x,
         > **NOTE** This is a safety mechanism against excessive slippage. The [`preview_swap_exact_output()`](#preview_swap_exact_output) function can be used to calculate a reasonable maximum input amount.    
     8. Requires a deadline (block height limit)
 
-#### Previews
+### Previews
 
-##### `preview_add_liquidity()`
+#### `preview_add_liquidity()`
 
 1. Returns the amount of the other asset to input and the liquidity asset amount to receive after an add liquidity operation
     1. If the asset pair of the pool is set 
     2. Requires the amount of an asset to input to [`add_liquidity`](#add_liquidity) 
         > **NOTE** If any liquidity in the contract already exists, than the amount of the other asset is calculated based on the ratio of the assets. Otherwise, the ratio is assumed to be 1.   
 
-##### `preview_swap_exact_input()`
+#### `preview_swap_exact_input()`
 
 1. Returns the minimum output amount to receive after a [`swap_exact_input`](#swap_exact_input) and whether the output asset reserves are sufficient for the swap
     1. If the asset pair of the pool is set 
     2. Requires an exact amount of either asset to sell
 
-##### `preview_swap_exact_output()`
+#### `preview_swap_exact_output()`
 
 1. Returns the maximum input amount for a [`swap_exact_output`](#swap_exact_output) and whether the input asset reserves are sufficient for the swap
     1. If the asset pair of the pool is set
     2. If the output asset reserves are sufficient for the swap
     3. Requires an exact amount of either asset to buy
 
-#### State Checks
+### State Checks
 
-##### `balance()`
+#### `balance()`
 
 1. Returns the asset balance of the sender in the contract 
     1. If the asset pair of the pool is set
     2. Requires the asset identifier to return the balance for
 
-##### `pool_info()`
+#### `pool_info()`
 
 1. Returns the pool info, i.e., the identifiers and amounts of assets and the liquidity pool asset amount 
     1. If the asset pair of the pool is set
 
-## Sequence Diagram
+## Atomic Add Liquidity Script
```

Comment:
How about a `### Script` heading and then nesting the scripts in there? You could exclude the `Script` post-fix and have `Atomic Add Liquidity` etc. or better yet the name of the script itself to follow the convention of the functions above.

Reply:
Done.